pages/welcome.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import Container from '../components/Container';
+import { Box, Flex } from '../components/Grid';
+import Image from '../components/Image';
+import Link from '../components/Link';
+import Page from '../components/Page';
+import StyledCard from '../components/StyledCard';
+import StyledLink from '../components/StyledLink';
+import { useUser } from '../components/UserProvider';
+
+const WelcomeOptionContainer = styled(Container)`
+  &:hover {
+    background-color: #f5faff;
+  }
+`;
+
+const Welcome = () => {
+  const { LoggedInUser, loadingLoggedInUser } = useUser();
+  const router = useRouter();
+  if (!loadingLoggedInUser && !LoggedInUser) {
+    router.push('/');
+  }
```

Comment:
Redirect is inelegant, the user should see a sign in form if they're not authenticated. `AuthenticatedPage`

Reply:
Makes sense, I've added a new page called `AuthenticatedPageV2.js` which is similar to `AuthenticatedPage.js` but uses the new `SignInOrJoinFreeV2.js` page. And used this `AuthenticatedPageV2.js` here. We'll need this `AuthenticatedPageV2.js` in the future to migrate exiting situations like these when users log out a page where they are supposed to be logged in to view. Anyways, with this addition when the user logs out of the `/welcome` page they see,

![image](https://user-images.githubusercontent.com/12435965/172938439-13e62687-429e-44b5-bc51-b33825cf8a6c.png)

Reply:
A simple prop would have been enough (`<AuthenticatedPage formVersion="v2" />`). Or using the default form, since we'll replace it globally when releasing the project anyway.

But that's not a blocker.

Reply:
I'll keep this change as is for now, I thought about adding a prop and going with this prop method, but somehow I preferred this `AuthenticatedPageV2` approach better as I thought it's easy to just remove the older component and rename the new one when we are releasing rather than adjusting the props.